<launch>
    <arg name="points_parent_src" default="/vlp16_left/velodyne_points" />
    <arg name="points_child_src" default="/vlp16_right/velodyne_points" />
    <arg name="voxel_size" default="1.0" />
    <arg name="ndt_epsilon" default="0.01" />
    <arg name="ndt_step_size" default="0.1" />
    <arg name="ndt_resolution" default="1.0" />
    <arg name="ndt_iterations" default="400" />
    <arg name="x" default="0" />
    <arg name="y" default="-2.6" />
    <arg name="z" default="0" />
    <arg name="roll" default="0" />
    <arg name="pitch" default="0" />
    <arg name="yaw" default="0" />
    <arg name="maxcount" default="50" />

    <node pkg="multi_lidar_calibrator" type="multi_lidar_calibrator" name="lidar_calibrator" output="screen">
        <param name="points_parent_src" value="$(arg points_parent_src)" />
        <param name="points_child_src" value="$(arg points_child_src)" />
        <param name="voxel_size" value="$(arg voxel_size)" />
        <param name="ndt_epsilon" value="$(arg ndt_epsilon)" />
        <param name="ndt_step_size" value="$(arg ndt_step_size)" />
        <param name="ndt_resolution" value="$(arg ndt_resolution)" />
        <param name="ndt_iterations" value="$(arg ndt_iterations)" />
        <param name="x" value="$(arg x)" />
        <param name="y" value="$(arg y)" />
        <param name="z" value="$(arg z)" />
        <param name="roll" value="$(arg roll)" />
        <param name="pitch" value="$(arg pitch)" />
        <param name="yaw" value="$(arg yaw)" />
	<param name="maxcount" value="$(arg maxcount)" />
    </node>

</launch>
